src/display/display_utils.js

```diff
@@ -589,12 +589,25 @@ function getRGB(color) {
   return [0, 0, 0];
 }
 
+function getColorValues(colors) {
```

Comment:
Are you expecting to use this anywhere *except* in the Editor-layer?
If not, should we perhaps just place this code in the `src/display/editor/tools.js` file instead?

Reply:
I thought it could be reused to get the real colors we pass to the canvas in HCM.
We do something similar here:
https://github.com/mozilla/pdf.js/blob/ce63663282e9f7fff4adecf11cc030f436d7d4cc/src/display/canvas.js#L1291-L1292